src/ext/java/org/opentripplanner/ext/examples/updater/ExampleGraphUpdater.java

```diff
@@ -41,43 +42,41 @@
     private String url;
 
     // Here the updater can be configured using the properties in the file 'Graph.properties'.
-    @Override
-    public void configure(Graph graph, JsonNode config) throws Exception {
-        frequencySec = config.path("frequencySec").asInt(5);
-        url = config.path("url").asText();
+    public void configure(Graph graph, ExampleGraphUpdaterConfig config) {
+        frequencySec = config.getFrequencySec();
+        url = config.getUrl();
         LOG.info("Configured example updater: frequencySec={} and url={}", frequencySec, url);
     }
 
     // Here the updater gets to know its parent manager to execute GraphWriterRunnables.
-    @Override
     public void setGraphUpdaterManager(GraphUpdaterManager updaterManager) {
         LOG.info("Example updater: updater manager is set");
         this.updaterManager = updaterManager;
     }
 
     // Here the updater can be initialized.
-    @Override
     public void setup(Graph graph) {
         LOG.info("Setup example updater");
     }
 
     // This is where the updater thread receives updates and applies them to the graph.
     // This method only runs once.
-    @Override
     public void run() {
         LOG.info("Run example updater with hashcode: {}", this.hashCode());
         // Here the updater can connect to a server and register a callback function
         // to handle updates to the graph
     }
 
     // Here the updater can cleanup after itself.
-    @Override
     public void teardown() {
         LOG.info("Teardown example updater");
     }
 
-    @Override
     public String getName() {
         return "ExampleGraphUpdater";
     }
+
+    public interface ExampleGraphUpdaterConfig
+        extends PollingGraphUpdater.PollingGraphUpdaterConfig {
+    }
```

Comment:
Tip! There is no need to extend - unless you add/override parameters. The good thing of not extending is that is is clear that the super-config is enough.

Reply:
Also, this break the established naming convention - using `Parameters` for the role defined by the config consumer, and using `Config` for the role implementing the `Parameters` delivering the config to the consumer/domain.